client/src/actions/actions.js

```diff
@@ -78,22 +95,75 @@ function userCreated() {
   }
 }
 
-export function createUser(firstName, lastName, phone, email, password) {
+export function createUser(user) {
```

Comment:
Could be a good opportunity to use destructuring here (on the user parameter, so you don't have to write out as much in the request body).